tests/search/vespa/test_vespa_concept_filters.py

```diff
@@ -0,0 +1,86 @@
+from unittest.mock import patch
+
+import pytest
+
+from app.api.api_v1.routers import search
+from app.api.api_v1.schemas.search import SearchResponse
+from tests.search.vespa.setup_search_tests import (
+    _make_search_request,
+    _populate_db_families,
+)
+
+
+@pytest.mark.search
+@patch("app.api.api_v1.routers.search.verify_any_corpora_ids_in_db", return_value=True)
+@pytest.mark.parametrize(
+    "label,query,concept_filters",
+    [
+        ("search", "the", [{"name": "name", "value": "environment"}]),
+        (
+            "search",
+            "the",
+            [
+                {"name": "parent_concept_ids_flat", "value": "Q0,"},
+                {"name": "id", "value": "0"},
+            ],
+        ),
+        (
```

Comment:
@katybaulch I'm getting some odd behaviour in the tests and wondering whether anything jumps out to you. 

For this third parameterized test input I'm receiving the following error: 

> (Pdb) response 
<Response [400 Bad Request]>
(Pdb) response.text
'{"detail":"Invalid Query: Failed to build query: Invalid query parameter"}'

If I then set a query string I receive the following error: 

> >>> response 
> <Response [400 Bad Request]>
> >>> response.text
> '{"detail":"Could not decode configuration token"}'

Reply:
tests/search/vespa/test_vespa_concept_filters.py::test_concept_filters[search-the-concept_filters0] PASSED                      [ 33%]
tests/search/vespa/test_vespa_concept_filters.py::test_concept_filters[search-the-concept_filters1] FAILED                      [ 66%]